packages/net/src/generated/netConfigData.ts

```diff
@@ -1,38 +1,28 @@
 export const netConfigData = {
-  "mainnet": {
-    "version": "0.38.0",
-    "apiUrls": [
+  mainnet: {
+    version: "0.38.0",
+    apiUrls: [
       "https://api.akashnet.net:443",
       "https://akash-api.polkachu.com:443",
       "https://akash.c29r3.xyz:443/api",
       "https://akash-api.global.ssl.fastly.net:443"
     ],
-    "rpcUrls": [
+    rpcUrls: [
       "https://rpc.akashnet.net:443",
       "https://rpc-akash.ecostake.com:443",
       "https://akash-rpc.polkachu.com:443",
       "https://akash.c29r3.xyz:443/rpc",
       "https://akash-rpc.europlots.com:443"
     ]
   },
-  "sandbox": {
-    "version": "0.38.0",
-    "apiUrls": [
-      "https://api.sandbox-01.aksh.pw:443"
-    ],
-    "rpcUrls": [
-      "https://rpc.sandbox-01.aksh.pw:443"
-    ]
+  sandbox: {
+    version: "0.38.0",
+    apiUrls: [""],
```

Comment:
Something is wrong here and could the cause of the tests failing

Reply:
Yes... I didn't notice these changes, these were done by a script during commit:

```
> @akashnetwork/net@0.0.2 generate
> ts-node scripts/generate.ts

Generate network configuration: mainnet, sandbox, testnet-02
Network configuration was written to ./packages/net/src/generated/net-config.ts
```

I made another commit and now the file looks to be the same, apart from formatting changes